native/cocos/renderer/gfx-vulkan/states/VKSampler.cpp

```diff
@@ -42,16 +42,20 @@ CCVKSampler::CCVKSampler(const SamplerInfo &info) : Sampler(info) {
     _gpuSampler->addressW = _info.addressW;
     _gpuSampler->maxAnisotropy = _info.maxAnisotropy;
     _gpuSampler->cmpFunc = _info.cmpFunc;
+    _gpuSampler->init();
+}
+
+CCVKSampler::~CCVKSampler() {
+    _gpuSampler = nullptr;
+}
```

Comment:
I think it is not needed to implement destructor like this. Can just use default implementation.